### PR TITLE
Assignment 4 Done

### PR DESCRIPTION
**Pull request by AbhishekPandey-successive**

### CODE REVIEW

**Comment by AbhishekPandey-successive** [comment]
I have remove duplicates the Duplicates by using forEach method, not used set.

File: Assignment4/Question10.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Question 10
+
+let array1=[1,2,3,4,5];
+let array2=[6,7,8,9,10];
+
+let array3=array1.concat(array2);
```
Comment (Rohit0203) [inline Assignment4/Question10.js:6]: you can use spread operator also

**Review by Rohit0203 — APPROVED** [review]
Good

File: Assignment4/Question10.js (hunk 1 above)
Comment (AbhishekPandey-successive) [reply Assignment4/Question10.js:6]: yes it is Also Alternative way.

**Pull request merged**